scripts/onebranch/post-build.ps1

```diff
@@ -12,21 +12,40 @@ Set-Location $scriptPath\..\..
 $OneBranchArch = $env:ONEBRANCH_ARCH
 $OneBranchConfig = $env:ONEBRANCH_CONFIG
 
+function Copy-BuildFolder {
+    param (
+        [Parameter(Mandatory=$true)]
+        [ValidateSet("Release", "Debug")]
+        [string]$Configuration
+    )
+
+    # Define the source folder and zip file path based on the configuration
+    $SourceFolder = ".\x64\$Configuration"
```

Comment:
I think the NTOS repo doesn't support arm, does it? In any case - you may want to consider generalizing the arch as an input parameter to make this a bit more future proof. I think we will want to support arm eventually.

Reply:
added